.dev/assets/shared/css/amp/amp.css

```
@@ -0,0 +1,28 @@
+body[class*="amp"] .primary-menu .menu-item-has-children:hover > .sub-menu {
+	display: none;
+}
+
+body[class*="amp"] .primary-menu .menu-item-has-children:hover button.dropdown-toggle.toggled-on + ul,
+body[class*="amp"] .primary-menu button.toggled-on + ul {
+	display: block;
+}
+
+body[class*="amp"] .primary-menu button.dropdown-toggle {
+	background: transparent;
+	left: 0;
+	position: absolute;
+	top: 0;
+	width: 100%;
+}
+
+body[class*="amp"] .primary-menu .sub-menu a {
+	color: inherit;
+}
+
+body[class*="amp"] .header__navigation a {
+	color: hsla(var(--theme-site-nav--color), 1);
+}
+
+.primary-menu .sub-menu {
+	background-color: hsl(var(--theme-submenu--bg, var(--theme-color-heading)));
```

This value is getting dropped when parsed by [PHP-CSS-Parser](https://github.com/sabberworm/PHP-CSS-Parser).

A workaround that is working for me is to store the `hsa()` value in the variable, rather than the arguments that go into the `hsa()` function. So like this:

```css

.primary-menu .sub-menu {
    background-color: var(--theme-submenu--bg, var(--theme-color-heading));
}
```

Something like this:

```css
:root {
    --theme-header--bg: hsl(236, 47%, 46%);
}
/* ... */
.primary-menu .sub-menu {
    background-color: var(--theme-submenu--bg, var(--theme-color-heading));
}
```


```suggestion
	background-color: var(--theme-submenu--bg, var(--theme-color-heading));
```